crates/cli/tests/reference/getter-setter.wat

```diff
@@ -11,16 +11,16 @@
   (type (;9;) (func (param i32 f64)))
   (import "./reference_test_bg.js" "__wbindgen_init_externref_table" (func (;0;) (type 0)))
   (func $__wbindgen_realloc (;1;) (type 8) (param i32 i32 i32 i32) (result i32))
-  (func $__wbindgen_malloc (;2;) (type 6) (param i32 i32) (result i32))
```

Comment:
I am not sure why I needed these changes. I am using rustc 1.86.0, maybe I should be using another version?